src/NuGet.Core/NuGet.Packaging/GlobalSuppressions.cs

```diff
@@ -217,16 +217,9 @@
-[assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Rules.ReferencesInNuspecMatchRefAssetsRule.GetReferencesFromNuspec(System.IO.Stream)~System.Collections.Generic.IDictionary{System.String,System.Collections.Generic.IEnumerable{System.String}}")]
-*/
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.PackageBuilder.CreatePart(System.IO.Compression.ZipArchive,System.String,System.IO.Stream,System.DateTimeOffset,System.Text.StringBuilder)")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.PackageBuilder.ValidateFilesUnique(System.Collections.Generic.IEnumerable{NuGet.Packaging.IPackageFile})")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Rules.DependenciesGroupsForEachTFMRule.Categorize(System.Collections.Generic.IEnumerable{System.String},System.Collections.Generic.IEnumerable{NuGet.Frameworks.NuGetFramework})~System.ValueTuple{System.Collections.Generic.HashSet{NuGet.Frameworks.NuGetFramework},System.Collections.Generic.HashSet{NuGet.Frameworks.NuGetFramework},System.Collections.Generic.HashSet{NuGet.Frameworks.NuGetFramework}}")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Rules.ReferencesInNuspecMatchRefAssetsRule.GetReferencesFromNuspec(System.IO.Stream)~System.Collections.Generic.IDictionary{System.String,System.Collections.Generic.IEnumerable{System.String}}")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Signing.IntegrityVerificationProvider.VerifyPackageIntegrityAsync(NuGet.Packaging.Signing.ISignedPackageReader,NuGet.Packaging.Signing.PrimarySignature,NuGet.Packaging.Signing.SignedPackageVerifierSettings)~System.Threading.Tasks.Task{NuGet.Packaging.Signing.PackageVerificationResult}")]
+[assembly: SuppressMessage("Style", "IDE0051:Remove unused private members", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.PackageDependencyGroup.#ctor(NuGet.Frameworks.NuGetFramework)")]
```

Comment:
Similar to another comment, why wasn't this method deleted, and if it shouldn't be deleted, can we use an inline suppression instead?